### ISSUE TITLE
Released form keeps on saving data

### ISSUE THREAD

**Issue by fzaninotto** [simsalabim/sisyphus#83]
Here is my situation. I have a server-side data transformer which takes YouTube URLs, and transforms them into YouTube iframes. This way, users can add an URL in a textarea and get a nice video on the resulting page. 

Sisyphus breaks this. The reason? When a user submits the form, the form is released, but the listeners aren't removed. As a consequence, the form continues to save data. I have configured a short timeout for sisyphus (a few seconds) and the server page takes a few seconds to response. Most of the time (but not always), the textarea is saved again _after submission_. So even if the server returns an iframe instead of a URL, the next time the form appears, sisyphus reloads the saved data and overrides the iframe with the url. The data is saved correctly, but the display to the user is wrong.

Long story short, listeners on a form must be removed when a form is released.
